### ISSUE TITLE
Trying to use without create-react-app

### ISSUE THREAD

**Issue by khushhal** [mzohaibqc/antd-live-theme#6]
I added the config-override.js, variable.less  and also index.less file.

and in webpack i added this configuration

`const options = {
  stylesDir: path.join(__dirname, './../'),
  antDir: path.join(__dirname, './../node_modules/antd'),
  varFile: path.join(__dirname, './../variables.less'),
  mainLessFile: path.join(__dirname, './../index.less'),
  themeVariables: ['@primary-color', '@text-color'],
}

const themePlugin = new AntDesignThemePlugin(options);`

but on running the development webpack server i am getting this error

node_modules/antd/lib/style/index.less in Missed semicolon
You tried to parse Less with the standard CSS parser; try again with the postcss-less parser.

can you point out is i am doing something wrong or is there any issue with using without create-react-app boilerplate.

**Comment by khushhal** [comment]
Yeah it correct, the webpack file is not in root directory its under config folder. is it required to webpack file in root directory ?

**Comment by mzohaibqc** [comment]
@khushhal It should be in root directory
